text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,329 @@
+
+1. CDK Bootstrap with default stack set roles
+   * Bootstrap parent account
+      * `cdk bootstrap aws://11111111111/us-east-1 --stack-set`
+      * The region here is where the stack set will be created in the parent account.
+   * Bootstrap child account
+      * `cdk bootstrap aws://22222222222/us-east-1
+            --stack-set
+            --trust 11111111111
+            --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess`
+      * The child account **must** be bootstrapped to the same region where the parent account is bootstrapped.
+      Although bootstrapping must be in the same region,
+      stack set instances can still be provisioned in any region of the same AWS partion in the child account.
+   * Specifying custom qualifier:
+      * `cdk bootstrap aws://11111111111/us-east-1 --qualifier xyz`
+      * `cdk bootstrap aws://22222222222/us-east-1
+            --trust 11111111111
+            --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess
+            --qualifier xyz`
+      * Qualifier must match in both the parent and child accounts.
+2. Manually create roles
+   * AWS Guide: [Grant self-managed permissions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs-self-managed.html)
+      * The AWS documentation will guide you through creating the Stack Set Administration role for the parent account,
+      and the Stack Set Execution roles for the child accounts.
+   * The administration role and execution role name should be specified as part of the StackSet properties.
+
+#### Usage
+
+To define a stack set, extend the core construct `StackSet`.
+Then define AWS resources in the same way as defining resources for stacks.
+Compared to the `Stack` construct, the `StackSet` core construct has different properties specific to StackSets.
+The properties define the configuration of the stack set and the preferences used for update operations.
+
+**Example:**
+
+```ts
+import * as cdk from '@aws-cdk/core';
+import * as iam from '@aws-cdk/aws-iam';
+
+class PocStackSet extends cdk.StackSet {
+    constructor(scope: cdk.Construct, id: string, props?: cdk.StackSetProps) {
+        super(scope, id, props);
+
+        new iam.Role(this, 'ExampleRole', {
+          assumedBy: new iam.ServicePrincipal('s3.amazonaws.com')
+        });
+    }
+}
+
+// Using default cdk bootstrap stack set roles and default qualifier
+new PocStackSet(app, 'poc-stack-set-1', {
+  faultTolerancePercentage: 10,
+  maxConcurrentPercentage: 10,
+  regionConcurrencyType: cdk.StackSetRegionConcurrency.PARALLEL,
+  //...
+});
+
+// Using default cdk bootstrap stack set roles with custom qualifier
+new PocStackSet(app, 'poc-stack-set-2', {
+  qualifier: 'xyz',
+  faultTolerancePercentage: 10,
+  maxConcurrentPercentage: 10,
+  regionConcurrencyType: cdk.StackSetRegionConcurrency.PARALLEL,
+  //...
+});
+
+// Using customized stack set roles
```

Comment:
To maintain a good flow for the document, I would put all the information about custom bootstrapping, roles and qualifiers to a separate section at the end "Customizing StackSet Bootstrapping".